pkg/controller/apmserver/apmserver_controller_test.go

```diff
@@ -28,7 +29,9 @@ import (
 
 var certSecretName = "test-apm-server-apm-http-certs-internal" // nolint
 
-type testParams = DeploymentParams
+type testParams struct {
```

Comment:
Is this just in case we want to add more params later?

Reply:
I cannot use `type testParams = deployment.Params`, types definitions like that can only work if the type on the right is local to the package :/
Struct embedding is the closest thing I found to keep a similar behaviour.

Reply:
Oh I missed that right below we were using it as a receiver, that makes sense now. I thought we were just using it as a parameter and didn't need an extra struct at all.